examples/flowertune-llm/pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "flowertune-llm"
+version = "1.0.0"
+description = ""
+license = "Apache-2.0"
+dependencies = [
+    "flwr[simulation]>=1.10.0",
+    "flwr-datasets>=0.3.0",
+    "trl==0.8.1",
+    "bitsandbytes==0.43.0",
+    "scipy==1.13.0",
+    "peft==0.6.2",
+    "fschat[model_worker,webui]==0.2.35",
+    "transformers==4.39.3",
+    "sentencepiece==0.2.0",
+    "omegaconf==2.3.0",
+    "hf_transfer==0.1.8",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "flwrlabs"
+
+[tool.flwr.app.components]
+serverapp = "flowertune_llm.server_app:app"
+clientapp = "flowertune_llm.client_app:app"
+
+[tool.flwr.app.config]
+dataset.name = "vicgalle/alpaca-gpt4"
+model.name = "openlm-research/open_llama_7b_v2"
```

Comment:
Would it be better to have as default the 3B model? In this way, we lower the barrier for running the example with default config. We discussed this a while back but I don't remember what we decided...

Reply:
Yeah let's use the 3B model as default then